capstone_project/src/main/java/com/google/sps/servlets/PostServlet.java

```diff
@@ -52,6 +60,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     PostService postService = PostService.Builder.builder().build();
     postService.storePost(request);
+    PostAnalysis postAnalysis = new PostAnalysis(request);
+
+    request.getSession().setAttribute("categories", postAnalysis.getCategories());
```

Comment:
shouldn't this be happening before you store the post?

Reply:
I don't actually store the post, but I need to create the PostAnalysis to classify the categories, line 65 then stores the categories in the request so that I can access it in comments.jsp when the page is refreshed!

Reply:
Oic. Can you confirm that you mean this to only show the resources WHEN the post gets posted? IE if people look at the post in the future they wont see the resources?